test/cluster/object_store/test_backup.py

```diff
@@ -373,24 +373,18 @@ def __init__(self, rf, nodes, racks, dcs):
         self.racks = racks
         self.dcs = dcs
 
-@pytest.mark.asyncio
```

Comment:
I guess this patch doesn't just "adds test", but refactors the existing one prior to that, so that new test could re-use code blocks from the old one. If so, please, split this patch respectively.

Reply:
I opened https://github.com/scylladb/scylladb/pull/24105 to merge it separately, I'll rebase the changes here once we merge that.